File: js/scraper.js
url = "https://www.whats-on-netflix.com/library/categories/";

function outputName() {
    fetch(url)
	.then((response) => {
  		return response.text();
	})
	.then((text) => {
        console.log(text);
        //document.getElementById('randomName').innerHTML = randomName;
	});
};

File: js/scraper.py
import requests
from bs4 import BeautifulSoup

URL = "https://www.whats-on-netflix.com/library/categories/"
page = requests.get(URL)
soup = BeautifulSoup(page.text,'html.parser')
results = soup.find(id="devices")


#print(results.text)
with open('movies.txt', 'w') as f:
    f.write(results.text.strip())